diesel_codegen/src/queryable.rs

```diff
@@ -41,7 +41,7 @@ pub fn expand_derive_queryable(
 
         let impl_item = quote_item!(cx,
             impl$generics ::diesel::Queryable<__ST, __DB> for $ty where
```

Comment:
This double underscoring squicks me out. I see this is in codegen so maybe it's conventional? It's applied consistently, so maybe no change here is necessary?